fsharp-backend/tests/Tests/Execution.Tests.fs

```diff
@@ -83,7 +83,7 @@ let testExecFunctionTLIDs : Test =
 
 let testErrorRailUsedInAnalysis : Test =
   testTask
-    "When a function which isn't available on the client has analysis data, we need to make sure we process the errorrail functions correctly" {
+    "When a function which isn't available on the client, but has analysis data, we need to make sure we process the errorrail functions correctly" {
```

Comment:
not sure if this is the right interpretation, but the title didn't quite make sense to me

Reply:
This is right. The "which" is no longer needed here though.